boards/DIM/Test/Src/Test_RegenPaddle.cpp

```diff
@@ -0,0 +1,63 @@
+#include "Test_Dim.h"
+
+extern "C"
+{
+#include "App_RegenPaddle.h"
+}
+
+FAKE_VALUE_FUNC(float, get_paddle_position);
+
+class RegenPaddleTest : public DimTest
+{
+  protected:
+    void SetUp() override
+    {
+        regen_paddle = App_RegenPaddle_Create(
+            get_paddle_position, DEFAULT_LOWER_DEADZONE,
+            DEFAULT_UPPER_DEADZONE);
+    }
+
+    void TearDown() override
+    {
+        ASSERT_TRUE(regen_paddle != NULL);
+        App_RegenPaddle_Destroy(regen_paddle);
+        regen_paddle = NULL;
+    }
+    const float         DEFAULT_LOWER_DEADZONE = 5.0f;
+    const float         DEFAULT_UPPER_DEADZONE = 95.0f;
+    struct RegenPaddle *regen_paddle;
+};
+
+TEST_F(RegenPaddleTest, check_lower_deadzone)
+{
+    get_paddle_position_fake.return_val = std::nextafter(
```

Comment:
using `nextafter` as a consistent way to test "threshold value - epsilon", where epsilon is a small change in value.